Tech Journey/My-#100DayOfCode-Journey-So-Far.md

```diff
@@ -0,0 +1,56 @@
+## A bit about me, Who am I?
+Hey, my name is **Ameen**, I am 15 years old and currently in high school. I started learning to code when I was in second grade in school but never took it seriously until I was 11... That time I started learning **Unity** ( A Game Engine ) and **C#** but due to my PC's specs I couldn't continue on that path so instead of giving up  I started learning **Python**, learned **PyGame** and a few more libs and built something basic with each of em.
+But I didn't enjoy it as much and started to lose my interest in python. So the hunt to find something fun had begun again. After exploring for a while, I found "**Web Development**", when I started learning web development it just clicked with me didn't take me much time to master HTML but then school came in the way . 2 years later I again continued the web dev path with the #100DaysOfCode challenge and am having fun ever since.
+I currently know HTML, CSS, js, and learning svelte and built around 12-18 projects alongside with it. That's enough about me let's get into the challenge.
+<hr>
+## What is **#100DaysOfCode** ??
+**#100DaysOfCode** is a challenge where you have to code ** 1-hour** at minimum for **100** days straight. The only reason you can take a break is if you are sick, need to travel, or just want to relax or take a break, just try not to take a break for more than 2 days at once.
+That's pretty much it plans to master something and code for 100 days straight and even tweet your progress on twitter :)
+
+[Official Site](https://www.100daysofcode.com/) for the challenge.
+The dev community on twitter is very supportive and welcoming. 
+<hr>
+## How I discovered the #100DaysOfCode Challenge?
+My summer vacation had started, and I was going through some blog posts on FreeCodeCamp and read about the **#100DaysOfCode** and thought to myself what if I do the challenge ...  being the guy I'm procrastinated for a week and then finally started the challenge on 27 June missed a few days due to traveling and hit 50 days milestone on 18 of August.
+ 
+##### My first day : 
+%[https://twitter.com/crafter_coder/status/1276863273003089921?s=19]
+
+## The Resources I used :
+
+I have used **many** resources, I will be sharing the one's which had the **maximum impact** on me as well proved to be very useful ...
+
+
+### [FreeCodeCamp](https://www.freecodecamp.org/learn) 
+
+How much ever I praise the site .. it will be less . If you want to learn web-development this should be your goto site. It's amazing and has a great supportive community. When I wasn't learning I would be scrolling through their forums and answer/advice any question if I knew about that subject .. it helped me to revise the concepts and stay on track. Their blog posts are amazing and informative too... So if you are a beginner and want to learn web development than this should be your goto place.
```

Comment:
"How much ever I praise the site .. it will be less ." -- avoid triple dot, can be said as "Any praise won't be enough for the makers of freeCodeCamp"